packages/okta-react/src/AuthService.js

```diff
@@ -94,32 +107,55 @@ class AuthService {
 
   emitAuthState(state) { 
     this._authState = state;
-    this.emit('authStateChange', this._authState);
+    this.emit('authStateChange', this.getAuthState());
```

Comment:
Should we scope the emitted event with `okta`?

Reply:
As this event will be emitted on `authService`, which you got from either `useOktaAuth` or `withOktaAuth`, this falls in the pattern of having Okta at the highest level but not redundantly at every level.